packages/fxa-settings/src/models/AlertBarInfo.ts

```diff
@@ -28,9 +28,10 @@ export class AlertBarInfo {
     alertContent(text);
   }
 
-  success(message: string) {
+  success(message: string, gleanEvent?: () => void) {
```

Comment:
I think this is fine, but I think might be nice to have a `setMetric` property that gets called on `this.show`. Maybe a follow up.

Reply:
Sounds good, will file a follow up for this!